### DISCUSSION TITLE
数字输入框的type为number类型和text类型的宽度不一样

### DISCUSSION THREAD

**Discussion by kelly2977** [layui/layui#2028] [问答]
![image](https://github.com/layui/layui/assets/50442723/156a488f-e4a3-4733-95b4-e0115f67184f)
![image](https://github.com/layui/layui/assets/50442723/4a12ab3d-4776-4b51-a0b0-7181bb5b1216)

这几种情况的数字输入框的type,max,class的不同，输入框的长度就明显不一样，不知道是什么情况，有大佬告知吗

**Comment by sentsim** [comment]
`layui-input-group` 中的宽度都是自由伸缩的，要保持一致的话，建议给 input 都设置个固定宽度。
